ironfish/src/account/accounts.ts

```diff
@@ -34,10 +34,6 @@ type SyncTransactionParams =
   | Record<string, never>
 
 export class Accounts {
-  readonly onDefaultAccountChange = new Event<
```

Comment:
This isn't consumed anywhere so removing it. I think this was only used by the old mining director code